.travis.yml

```diff
@@ -5,16 +5,25 @@ before_script:
 - yes | python setup.py install
 - git clone https://github.com/uc-cdis/dictionaryutils; cd dictionaryutils;
 script:
-- ./run_tests.sh
+- "./run_tests.sh"
 deploy:
-  provider: s3
-  access_key_id: AKIAIKM5YAHQGFDE72SQ
-  secret_access_key:
-    secure: ZTcjEHtJnAM39kLA55EiePoqZncTPeEZNTFX7E5xkc3TLAqgJlpjpoH40Qq7cn5gzdZ0Z9YWlOCyIV0Va6Oxc7XJVK/lSzioelseKnz+yscUq8aAJluy1INx7uXEbiPwRNz+bfy5uYOU/Bn/+6/I4Wc3MbvNRdbwF57hXdn6WjD7s+/T65l2PL3iN4MqLRmzuC+BRztOugnWM6s/za7pcA7AP9j367HX0yt/31h9CPIZuGFSItWjmxEFAB7VfCqN+fF7ZisOwGRDJkzzEv6eJbpsCq1m1Jl3TYfQoX96mUh0Pa2frk69enDVSsOkDeQ82DrREf51wdfk6Gs/YmrrVSoaBM0Cm97Ssd7fr2erxTlIdfRvPDN3vncWzsnvVZHkez2owqWV9TEaAgbVsI6myINAX0Ibh8beLkCHjFLXYoOBglNGrc2DC2M9ZBpM18nJ2tZVAva2t53MSFtcRw06xu4E9+ZBMZeVacXM1BA7M6APQVSDwrHJk3BVgaz+Wi+iUFZkmCujI5xS1PNta+iVZ6yJyiMd2l2oLsikcs9qYI94aZvmHYFSxq+LdZfIpov/10dPt800jHDgszKcGJ4rGGZOTFguI7fDPrkkTgihLthRuBGc5umMI4hdJZ4CB1JXCkWK/wk7hdBuMye7VzAEc9XfFflAUb+3H7T1PfhC+MM=
-  bucket: dictionary-artifacts
-  local-dir: artifacts
-  upload-dir: datadictionary/$TRAVIS_BRANCH
-  skip_cleanup: true
-  on:
-    repo: uc-cdis/datadictionary
-    all_branches: true
+  matrix:
+  - provider: s3
+    access_key_id: AKIAIKM5YAHQGFDE72SQ
+    secret_access_key:
+      secure: ZTcjEHtJnAM39kLA55EiePoqZncTPeEZNTFX7E5xkc3TLAqgJlpjpoH40Qq7cn5gzdZ0Z9YWlOCyIV0Va6Oxc7XJVK/lSzioelseKnz+yscUq8aAJluy1INx7uXEbiPwRNz+bfy5uYOU/Bn/+6/I4Wc3MbvNRdbwF57hXdn6WjD7s+/T65l2PL3iN4MqLRmzuC+BRztOugnWM6s/za7pcA7AP9j367HX0yt/31h9CPIZuGFSItWjmxEFAB7VfCqN+fF7ZisOwGRDJkzzEv6eJbpsCq1m1Jl3TYfQoX96mUh0Pa2frk69enDVSsOkDeQ82DrREf51wdfk6Gs/YmrrVSoaBM0Cm97Ssd7fr2erxTlIdfRvPDN3vncWzsnvVZHkez2owqWV9TEaAgbVsI6myINAX0Ibh8beLkCHjFLXYoOBglNGrc2DC2M9ZBpM18nJ2tZVAva2t53MSFtcRw06xu4E9+ZBMZeVacXM1BA7M6APQVSDwrHJk3BVgaz+Wi+iUFZkmCujI5xS1PNta+iVZ6yJyiMd2l2oLsikcs9qYI94aZvmHYFSxq+LdZfIpov/10dPt800jHDgszKcGJ4rGGZOTFguI7fDPrkkTgihLthRuBGc5umMI4hdJZ4CB1JXCkWK/wk7hdBuMye7VzAEc9XfFflAUb+3H7T1PfhC+MM=
+    bucket: dictionary-artifacts
+    local-dir: artifacts
+    upload-dir: datadictionary/$TRAVIS_BRANCH
+    skip_cleanup: true
+    on:
+      repo: uc-cdis/datadictionary
+      all_branches: true
+  - provider: pypi
+    user: uc-ctds
+    skip_existing: true
+    on:
+      tags: true
+      branch: develop
```

Comment:
Why `branch: develop`? I'm not sure how that behaves together with `tags: true`.

Reply:
We seem to be on `develop` by default (no master branch) and I just figured I'd make it explicit :P Seems like it doesn't matter since [all conditions must be met](https://docs.travis-ci.com/user/deployment#conditional-releases-with-on) to trigger a deployment, but I can definitely remove it too!

Reply:
Oh, that's good! Definitely keep it.